query-engine/driver-adapters/js/adapter-libsql/README.md

```diff
@@ -3,3 +3,93 @@
 Prisma driver adapter for Turso and libSQL.
 
 See https://prisma.io/turso for details.
+
+The following usage tutorial is valid for Prisma 5.4.2 and later versions.
+
+## How to install
+
+After [getting started with Turso](https://www.prisma.io/blog/prisma-turso-ea-support-rXGd_Tmy3UXX#create-a-database-on-turso), you can use the Turso serverless driver to connect to your database. You will need to install the `@prisma/adapter-libsql` driver adapter and the `@libsql/client` serverless driver.
+
+```sh
+npm install @prisma/adapter-libsql
+npm install @libsql/client
+```
+
+Make sure your Turso database connection string and authentication token is copied over to your `.env` file. The connection string will start with `libsql://`.
+
+```env
+# .env
+TURSO_AUTH_TOKEN="eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCJ9..."
+TURSO_DATABASE_URL="libsql://turso-prisma-random-user.turso.io"
+```
+
+You can now reference this environment variable in your `schema.prisma` datasource. Make sure you also include the `driverAdapters` Preview feature.
+
+```prisma
+// schema.prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["driverAdapters"]
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:./dev.db"
+}
+```
+
+Now run `npx prisma generate` to re-generate Prisma Client.
+
+## How to setup migrations
+
+As Turso needs to sync between a local sqlite database and another one hosted on Turso Cloud, an additional migration setup is needed. In particular, anytime you modify models and relations in your `schema.prisma` file, you should:
```

Comment:
The first sentence here is just wrong. There is no local SQLite database involved when using Prisma. There might be weird things needed for our migration flow as we do not support Migrations with Turso yet.